BMS Workspace-git/bms-1227-fw/Phantom_drivers/mock/mock_Phantom_sci.c

```diff
@@ -1,12 +1,12 @@
 #include <stdio.h>
 #include "sci.h"
 
-void UARTprintf(const char *_format, ...)
-{
-   return;
-}
-
-void UARTSend(sciBASE_t *sci, char data[])
-{
-	return;
-}
\ No newline at end of file
+//void UARTprintf(const char *_format, ...)
```

Comment:
Remove commented out code that won't be used in the future.

Reply:
- removed the mock folder, it contained mostly empty functions that were not in use in any files
- removed unused global variables from current_transducer.h